packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -270,6 +291,22 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
                 {title}
                 {callDescription}
               </Stack.Item>
+              {
+                /* @conditional-compile-remove(video-background-effects) */
+                <DefaultButton
+                  iconProps={{ iconName: 'OptionsVideoBackgroundEffect', styles: { root: { lineHeight: 0 } } }}
```

Comment:
store the styles in a styles file. This component will re-render needlessly if objects are not static or memoized.

Reply:
Looks like that styling is not needed, icon is rendering well without that as well, removed it.